src/App.tsx

```diff
@@ -1,26 +1,42 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { Route, Switch } from "react-router-dom";
+
+import Users from "./pages/Users";
+import UserPosts from "./pages/Posts";
+import Home from "./pages/Home";
+import UserTodos from "./pages/Todos";
+import UserForm from "./pages/UserForm";
+import PostForm from "./pages/PostForm";
+
+import "./App.css";
```

Comment:
We don't need css imports up til now

Reply:
I cannot understand this comment, so do I need to remove CSS imports?